docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -117,40 +120,86 @@ The Endpoint Detection Agent is easy to install on Windows or Linux systems once
     sudo systemctl status SEKOIAEndpointAgent.service
     ```
 
-Once installed, the agent collects, normalizes, and sends event logs to Sekoia.io. The protocol used to send events is HTTPS (443).
-
 #### journald configuration
 
 To get events, the agent pushes rules to the audit framework. By default, **journald**  might listen to the audit socket for events.
 
 To disable audit logging, do the following as root:
-	
-       
+
+
 	# Stop listening to audit events
 	systemctl stop systemd-journald-audit.socket 
-	
+
 	# Disable it to avoid future start
 	systemctl disable systemd-journald-audit.socket
-	
+
 	# Masking will prevent starting by other services
 	systemctl mask systemd-journald-audit.socket
-	
+
 	# Restart journald
 	systemctl restart systemd-journald
-       
-       
+
+
 A reboot may be necessary if the audit events are still appearing in the logs.
 
+=== "MacOs"
+
+    Move the application from its download folder to the /Applications directory.
+
+    Grant full disk access to SekoiaEndpointAgent by following these steps:
+
+        1.  Click on the Apple icon on the top left corner of your screen.
+
+        2.  Select System Preferences.
+
+        3.  Go to Security & Privacy Preferences > Privacy and click Full Disk Access from the left panel.
+
+        4.  Tick the checkbox for SekoiaEndpointAgent. If the lock at the bottom left is locked, click it to unlock the Privacy pane.
+
+        If you can't find SekoiaEndpointAgent in step 4, do the following:
+
+            1.  Click the plus (+) icon and navigate to Applications.
+
+            2.  In the left panel, find SekoiaEndpointAgent and click Open. Make sure SekoiaEndpointAgent is ticked and then close the window.
```

Comment:
Should not be displayed inside a code block.

Maybe it would be nice to have a gif or picture describing the operation. We can ask design later.